### DISCUSSION TITLE
HA Core Warnings and Errors

### DISCUSSION THREAD

**Discussion by tomhusband** [blakeblackshear/frigate#6088] [General]
```
<Unconnected @ 192.168.1.14> Trying again due to error: ConnectError('All connection attempts failed')

06:34:39 – (WARNING) components/amcrest/__init__.py - message first occurred at 05:49:39 and shows up 46 times

Error from stream worker: Error opening stream (HTTP_NOT_FOUND, Server returned 404 Not Found) http://192.168.1.15/9000

06:14:43 – (ERROR) Stream - message first occurred at 05:54:03 and shows up 2 times

HTTP error 404 Not Found

06:14:43 – (WARNING) components/stream/worker.py - message first occurred at 05:54:03 and shows up 2 times
```
I keep seeing these errors. The other day we had a power cut and when it was restored the IP address (192.168.1.14) for my one camera changed. I’ve since updated the Frigate config so everything is fine with Figate. I also made the new address static but I still get these errors referencing the old IP address. In devices I saw an Amcrest camera that said it was no longer used so I should delete it. I did but I still get the errors. Where can I delete the reference to it?
Many thanks.

**Comment by tomhusband** [comment]
There's also an error against the correct camera, http://192.168.1.15/9000, and I'm not sure what that's all about.

**Comment by NickM-27** [comment]
This looks entirely unrelated to frigate and not a frigate log

**Reply by tomhusband** [reply]
OK, sorry, I thought because it referenced the camera IP it had something to do with Figate. I'll look at the HA end of things.
